product-scenarios/4-single-sign-on/4.1-sso-for-web-app/4.1.1-sso-with-saml/src/test/java/org/wso2/identity/scenarios/sso/test/saml/SAMLSSOWithExternalAppTestCase.java

```diff
@@ -21,48 +21,33 @@
 import org.apache.http.Header;
 import org.apache.http.HttpHeaders;
 import org.apache.http.HttpResponse;
+import org.apache.http.HttpStatus;
 import org.apache.http.impl.client.CloseableHttpClient;
 import org.apache.http.message.BasicHeader;
 import org.apache.http.util.EntityUtils;
 import org.apache.xml.security.signature.XMLSignature;
+import org.json.simple.JSONArray;
+import org.json.simple.JSONObject;
 import org.testng.Assert;
 import org.testng.ITestResult;
-import org.testng.annotations.AfterClass;
-import org.testng.annotations.AfterMethod;
-import org.testng.annotations.BeforeClass;
-import org.testng.annotations.DataProvider;
-import org.testng.annotations.Factory;
-import org.testng.annotations.Test;
+import org.testng.annotations.*;
 import org.wso2.carbon.identity.sso.saml.stub.types.SAMLSSOServiceProviderDTO;
-import org.wso2.identity.scenarios.commons.SAML2SSOTestBase;
-import org.wso2.identity.scenarios.commons.SAMLConfig;
-import org.wso2.identity.scenarios.commons.SAMLSSOExternalAppTestClient;
-import org.wso2.identity.scenarios.commons.ScenarioTestBase;
-import org.wso2.identity.scenarios.commons.TestConfig;
-import org.wso2.identity.scenarios.commons.TestUserMode;
+import org.wso2.identity.scenarios.commons.*;
 import org.wso2.identity.scenarios.commons.clients.usermgt.remote.RemoteUserStoreManagerServiceClient;
 import org.wso2.identity.scenarios.commons.util.IdentityScenarioUtil;
 
 import java.util.HashMap;
 import java.util.Map;
 
-import static org.testng.Assert.assertTrue;
+import static org.testng.Assert.*;
```

Comment:
remove `*` and use exact classes

Reply:
Fixed with https://github.com/wso2/product-is/pull/14317/commits/1b4ab737ecfa0d7092d2c697a13b8ad39d9ad538